webrtc/videoroom/README.md

```diff
@@ -52,6 +52,36 @@ mix phx.server
 
 Then, go to <http://localhost:4000/>.
 
+### Run with docker
```

Comment:
Maybe add following sections
## Usage
### Environment Variables
Available runtime environmental variables:
```
VIRTUAL_HOST={host passed to the endpoint config, defaults to "localhost" on non-production environments}
USE_TLS={"true" or "false", if set to "true" then https will be used and certificate paths will be required}
KEY_FILE_PATH={path to certificate key file, used when "USE_TLS" is set to true}
CERT_FILE_PATH={path to certificate file, used when "USE_TLS" is set to true}
STUN_SERVERS={list of stun servers separated by ",", defaults to a single server "stun1.l.google.com:19302"}
TURN_SERVERS={list of turn servers separated by ",", defaults to ""}
```
### Run manually

Default path for certificate files for non-production environment is `priv/certs/`.
If you want to run TLS locally you can generate certificates
as described in the [signaling server readme](https://github.com/membraneframework/membrane_demo/tree/master/webrtc/simple#https). 
First install all dependencies:
```
mix deps.get
npm ci --prefix=assets
```
In order to run, type:
```
mix phx.server 
```

Then, go to <http://localhost:4000/>.

### Run with docker
...